### PR TITLE
Update links to reflect repository move to sqlparser-rs GitHub org

### PR DESCRIPTION
**Pull request by andygrove**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Version](https://img.shields.io/crates/v/sqlparser.svg)](https://crates.io/crates/sqlparser)
-[![Build Status](https://github.com/ballista-compute/sqlparser-rs/workflows/Rust/badge.svg?branch=main)](https://github.com/ballista-compute/sqlparser-rs/actions?query=workflow%3ARust+branch%3Amain)
-[![Coverage Status](https://coveralls.io/repos/github/ballista-compute/sqlparser-rs/badge.svg?branch=main)](https://coveralls.io/github/ballista-compute/sqlparser-rs?branch=main)
+[![Build Status](https://github.com/sqlparser-rs/sqlparser-rs/workflows/Rust/badge.svg?branch=main)](https://github.com/sqlparser-rs/sqlparser-rs/actions?query=workflow%3ARust+branch%3Amain)
+[![Coverage Status](https://coveralls.io/repos/github/sqlparser-r/sqlparser-rs/badge.svg?branch=main)](https://coveralls.io/github/sqlparser-r/sqlparser-rs?branch=main)
```
Comment (Dandandan) [inline README.md:6]: `sqlparser-rs`?

Comment (Dandandan) [inline README.md:6]: `sqlparser-rs`?

Comment (andygrove) [reply README.md]: Thanks. Fixed.

**Review by alamb — APPROVED** [review]
Looks good to me

**Pull request merged**